plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/file/ColumnStatistics.java

```diff
@@ -83,7 +83,10 @@ public HiveColumnStatistics toHiveColumnStatistics(HiveBasicStatistics basicStat
     {
         OptionalDouble averageColumnLength = this.averageColumnLength;
         if (totalSizeInBytes.isPresent() && basicStatistics.getRowCount().orElse(0) > 0) {
-            averageColumnLength = OptionalDouble.of(totalSizeInBytes.getAsLong() / (double) basicStatistics.getRowCount().getAsLong());
+            long nonNullCount = basicStatistics.getRowCount().getAsLong() - nullsCount().orElse(0);
```

Comment:
why `nullsCount().orElse(0)`? Seem we should not assume that there are no null if we don't know how many are there

Reply:
no strong opinion. changed